test/util/util.go

```diff
@@ -80,17 +80,14 @@ func DeleteNamespace(ctx context.Context, c client.Client, ns *corev1.Namespace)
 	if ns == nil {
 		return nil
 	}
-	err := c.DeleteAllOf(ctx, &batchv1.Job{}, client.InNamespace(ns.Name), client.PropagationPolicy(metav1.DeletePropagationBackground))
-	if err != nil && !apierrors.IsNotFound(err) {
-		return err
-	}
+	DeleteJobsInNamespace(ctx, c, ns)
```

Comment:
```suggestion
	err := DeleteJobsInNamespace(ctx, c, ns)
	if err != nil && !apierrors.IsNotFound(err) {
		return err
	}
```

Reply:
no need to check for IsNotFound anymore, but yes, I forgot to check if there is an error :)